src/main/scala/mesosphere/util/ConfigToMap.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.util
+
+import org.rogach.scallop.ScallopConf
+
+object ConfigToMap {
+
+  def convertToMap(config: ScallopConf): Map[String, Option[Any]] = {
```

Comment:
With the return type as written this method is unlikely to be useful anywhere else (we generally don't allow blind casts from `Any`).  It might just be better after all to move this code to a `Writes[ScallopConf]` in [Formats.scala](https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala).